themes/delphi/layouts/blog/list.html

```diff
@@ -1,6 +1,6 @@
-{{ define "breadcrumb" }}
+<!-- {{ define "breadcrumb" }}
   {{ partial "blog/breadcrumb.html" . }}
-{{ end }}
+{{ end }} -->
```

Comment:
issue: the blog main page and individual blog articles aren't loading in the preview (all other menu items and footer items load correctly). It looks to me like this is the only code change to the blog, so I'm wondering if this is causing the problem.

Reply:
Perhaps we could try commenting it out differently, with the comment inside the definition block instead of around it, as is done in: https://github.com/cmu-delphi/www-main/blob/3b554900b57476c159f8e71b04c51029ed15c9f1/themes/delphi/layouts/_default/covidcast_app.html#L7

Alternatively, if we got rid of the breadcrumbs to squash the weird "COVID-19" text that showed up on pages underneath the `epidemic-signals/` path, we can change that here: https://github.com/cmu-delphi/www-main/blob/5cc8c3ed1543bfebea6859954e87b2acba1425a8/content/epidemic-signals/_index.md?plain=1#L3

Reply:
@nmdefries @melange396 you both were right. Probably comment worked not as expected and caused wrong page rendering.
I don't understand why server does not show any issues/warning. Everything "works" properly, but blog and individual articles are not rendering at all. But sometimes that happens :+1: